x-pack/plugins/maps/public/classes/layers/wizards/layer_wizard_registry.ts

```diff
@@ -39,11 +39,13 @@ export type LayerWizard = {
   renderWizard(renderWizardArguments: RenderWizardArguments): ReactElement<any>;
   title: string;
   showFeatureEditTools?: boolean;
+  order?: number;
```

Comment:
Should `order` be required? Should we set order for existing maps wizards? Now order is registration order. Maybe this would be better coded with values. Maybe maps plugins should start at 100 and manually set numbers above that. Then plugins could use 0-99 to register before maps wizards ad 200+ to register after maps plugins.

Reply:
after discussion:

See proposal:
- 0-99: reserved for maps plugin
    - 10: maps-specific wizards
    - 20: solution layers
- 100+: non-Maps client plugins